astropy/tests/pytest_doctestplus.py

```diff
@@ -0,0 +1,341 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from ..extern import six
+
+import doctest
+import fnmatch
+import imp
+import os
+import re
+import sys
+
+import pytest
+
+from .output_checker import AstropyOutputChecker, FIX
+
+try:
+    import importlib.machinery as importlib_machinery
+except ImportError:  # Python 2.7
```

Comment:
@mohanagr , yes, if you want to take over this work from @Cadair , you need to open a new PR from your own branch in your own fork. However, if possible, please credit @Cadair for the initial work before you apply your own changes. One way is to follow guidance given at https://github.com/astropy/astropy/pull/3558#issuecomment-287369605 (and a few comments that followed) on how to cherry pick commits from this PR. Hope this helps.